aggsender/types/types.go

```diff
@@ -18,6 +17,56 @@ const (
 	AggchainProofMode    AggsenderMode = "AggchainProof"
 )
 
+type CertificateType string
+type CertificateTypeInt uint8
+
+const (
+	CertificateTypeUnknown CertificateType = ""
+	CertificateTypePP      CertificateType = "pp"
+	CertificateTypeFEP     CertificateType = "fep"
+
+	CertificateTypeUnknownInt CertificateTypeInt = 0
+	CertificateTypePPInt      CertificateTypeInt = 1
+	CertificateTypeFEPInt     CertificateTypeInt = 2
+)
+
+func (c CertificateType) String() string {
+	return string(c)
+}
+
+func (c CertificateType) ToInt() uint8 {
+	switch c {
+	case CertificateTypeFEP:
+		return uint8(CertificateTypeFEPInt)
+	case CertificateTypePP:
+		return uint8(CertificateTypePPInt)
+	default:
+		return uint8(CertificateTypeUnknownInt)
+	}
+}
+func NewCertificateTypeFromInt(i uint8) CertificateType {
+	switch i {
+	case uint8(CertificateTypePPInt):
+		return CertificateTypePP
+	case uint8(CertificateTypeFEPInt):
+		return CertificateTypeFEP
+	default:
+		return CertificateTypeUnknown
+	}
+}
+
+type CertificateSource string
+
+const (
+	CertificateSourceAggLayer CertificateSource = "agglayer"
+	CertificateSourceLocal    CertificateSource = "Local"
```

Comment:
To follow the same pattern:

```suggestion
	CertificateSourceLocal    CertificateSource = "local"
```